awslimitchecker/services/vpc.py

```diff
@@ -216,6 +217,33 @@ def _find_usage_nat_gateways(self, subnet_to_az):
                          'perhaps NAT service does not exist in this region?',
                          exc_info=1)
 
+    def _find_usage_peering_connections(self):
+        """find usage of Active VPC Peering Connections"""
+
+        peering_connections = self.conn.describe_vpc_peering_connections(
+            Filters=[{'Name': 'status-code', 'Values': ['active']}]
+        )['VpcPeeringConnections']
+
+        peering_connections_per_vpc = defaultdict(int)
+        for peering in peering_connections:
+            for vpc in [peering['AccepterVpcInfo'], peering['RequesterVpcInfo']]:
+                # Each peering contains 2 VPCs and at least one is in our current account and region. To avoid incorrect alerts about out-of-scope VPCs we need to filter to only VPCs in our account and region.
+
+                if vpc['OwnerId'] != self.current_account_id:
+                    continue
+
+                if vpc['Region'] != "us-west-2": # TODO: How to get current region?
```

Comment:
TODO: I'm not sure how to get the "active" region. Removing this check would result in VPCs being reported for multiple regions.

Reply:
I believe should be `self.conn._client_config.region_name` ?